video/out/gpu/context.c

```diff
@@ -37,7 +37,7 @@
 extern const struct ra_ctx_fns ra_ctx_glx;
 extern const struct ra_ctx_fns ra_ctx_x11_egl;
 extern const struct ra_ctx_fns ra_ctx_drm_egl;
-extern const struct ra_ctx_fns ra_ctx_cocoa;
+
```

Comment:
Stray newline?

Reply:
indeed, somehow failed to save the file properly.